rbi/sorbet/t.rbi

```diff
@@ -183,3 +183,11 @@ module T::Utils::Nilable
   def self.get_underlying_type(prop_type); end
   def self.get_underlying_type_object(prop_type); end
 end
+
+module T::LazyConstants
+  sig {params(val: BasicObject, klass: String).returns(T::Boolean)}
+  def self.lazy_is_a?(val, klass); end
+
+  sig {params(val: BasicObject, klass: String).returns(T::Boolean)}
+  def self.lazy_less_than(val, klass); end
```

Comment:
I don't want to introduce too much bikeshedding here, but what do you think about changing the name here? I'm a little worried that people will interpret it as value ordering, rather than a lazy version of `Module#<`.

Reply:
I'm actually at the point where I think we should probably bike shed, so bike shed away.

re this specific one, i might actually just delete this; not sure if i'll need it for what i'm working on. that being said, if you have a good name, i'm happy to keep it around.

Reply:
Maybe something like `lazy_module_less_than` or `lazy_module_subtype`? They're both gross long names, but maybe that'll make people think twice before relying on them? Also, deleting works too if you don't have a specific need for it now.

Reply:
👍 I'll delete for now and use something more similar to these names when I need them.